release_notes.md

```diff
@@ -8,3 +8,7 @@
 - Add function grouping information (https://github.com/Azure/azure-functions-host/pull/9735)
 - Bump `Microsoft.IdentityModel.Tokens`, `Microsoft.IdentityModel.Protocols.OpenIdConnect`, and
   `System.IdentityModel.Tokens.Jwt` from 6.32.0 to 6.35.0 (#9793)
+- Implement host configuration property for handling pre-cancelled invocation requests (#9523)
+  - If a worker supports CancellationTokens, cancelled invocations will now be sent to the worker by default
+    - Customers can opt-out of this behavior by setting `SendCanceledInvocationsToTheWorker` to `false` in host.json
```

Comment:
Do we need to update this page to reflect this new property? https://learn.microsoft.com/en-us/azure/azure-functions/functions-host-json.

Not sure we support a json schema for this file. If so, that also needs to be updated?

Reply:
I'm planning to make a PR to the schema and I think that auto updates this page